app/views/course/assessment/submission/submissions/_history.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+json.history do
```

Comment:
Do we still need this partial now?

Reply:
Yea, it will be rendered on `edit`, and pulls the past answer ids of each question, so we know if the question has any past answers or not.